animations/hero-animations/index.md

```diff
@@ -0,0 +1,670 @@
+---
+layout: page
+title: "Hero Animations"
+permalink: /animations/hero-animations/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* Create a Hero animation using Flutter's Hero widget.
+* Fly the hero from one screen to another.
+* Animate the transformation of a hero's shape from circular to
+  rectangular while flying it from one screen to another.
+* Hero animations are sometimes called _shared element transitions_
+  or _shared element animations_.
+</div>
+
+You've probably seen hero animations many times. For example,
+a screen displays a list of thumbnails representing items
+for sale.  Selecting an item flies it to a new screen,
+containing more details and a "Buy" button. Flying an image
+from one screen to another is called a _hero animation_
+in Flutter, though the same motion is sometimes referred to as
+a _shared element transition_.
+This guide demonstrates how to build standard
+hero animations, and hero animations that transform the
+image from a circular shape to a square shape during flight.
+
+* TOC Placeholder
+{:toc}
+
+<aside class="alert alert-info" markdown="1">
+**New to Flutter?**
+You should first understand how to create a layout using Flutter’s
+widgets.  For more information, see [Building Layouts in
+Flutter](/tutorials/layout/).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+A [_Route_](/routing-and-navigation/) describes a page or screen
+in a Flutter app.
+</aside>
+
+You can create this animation in Flutter with Hero objects.
+When a new route appears, a hero animates from the current route to
```

Comment:
I think it would be helpful to clarify that "appears" means that the rest of the route (less the Hero which is flying along its own path) animates into view. This is pretty plain to see in the video.